site/cypress/e2e/Cypress-Feature/forums.spec.js

```diff
@@ -11,6 +11,40 @@ const reply3 = 'Cypress Reply 3 Cypress';
 const merged1 = 'Merged Thread Title: '.concat(title3, '\n\n', content3);
 const merged2 = 'Merged Thread Title: '.concat(title2, '\n\n', content2);
 const attachment1 = 'sea_animals.png';
+const isVisible = 'be.visible';
```

Comment:
I don't agree with this. Why have it defined in one test and not others? I think this is odd stylistically, especially if it's only used in one test case.